tests/integration/update.bats

```diff
@@ -11,7 +11,7 @@ CGROUP_BLKIO=""
 
 function init_cgroup_path() {
     for g in MEMORY CPUSET CPU BLKIO; do
-        base_path=$(grep "rw,"  /proc/self/mountinfo | grep -i -m 1 "$g\$" | cut -d ' ' -f 5)
+        base_path=$(grep "rw,"  /proc/self/mountinfo | gawk 'toupper($10) ~ /\<'${g}'\>/ { print $5 }')
```

Comment:
Even more weirdly, the number changes between being in a container and being on the host. I'll use `$NF` instead then.